Assets/Scripts/Framework/Gameplay/Values/SessionValues.cs

```diff
@@ -0,0 +1,101 @@
+    /// this static class.
+    /// </summary>
+    public static class SessionValues
+    {
+        /// <summary>
+        /// The amount of time it takes for a note to enter the screen and exit the screen.
+        /// </summary>
+        public static float travelTime = SessionValuesShortcut.Instance.travelTime;
+        /// <summary>
+        /// The current time we are in the song, keeps being updated unless we are paused.
+        /// </summary>
+        public static float time = SessionValuesShortcut.Instance.travelTime * -1;
+
+        /// <summary>
+        /// The score that the player has.
+        /// </summary>
+        public static int score = SessionValuesShortcut.Instance.score;
+        /// <summary>
+        /// Multiplier for the score that is applied before adding to score.
+        /// </summary>
+        public static int scoreMultiplier = SessionValuesShortcut.Instance.scoreMultiplier;
+
+        /// <summary>
+        /// A bool that holds status for if the game is paused.
+        /// </summary>
+        public static bool paused = SessionValuesShortcut.Instance.paused;
+        /// <summary>
+        /// The currently selected Level ID.
+        /// </summary>
+        public static int currentLevelID = SessionValuesShortcut.Instance.currentLevelID;
+        /// <summary>
+        /// The currently selected Level, gets set when the ID is updated.
+        /// </summary>
+        public static LevelData currentLevel = SessionValuesShortcut.Instance.currentLevel;
+        /// <summary>
+        /// The selected difficulty, this is one value and if will be applied on the currentLevel when we start.
+        /// </summary>
+        public static Difficulty difficulty = SessionValuesShortcut.Instance.difficulty;
+        /// <summary>
+        /// The scriptableObject for all the levels, can be swapped out if necessary.
+        /// </summary>
+        public static Levels levels = SessionValuesShortcut.Instance.levels;
+
+        /// <summary>
+        /// The prefab for a note, can be any prefab if it has the Note.cs script.
+        /// </summary>
+        public static GameObject note = SessionValuesShortcut.Instance.note;
+
+        /// <summary>
+        /// Returns a bool for if the upcoming timestamp should be queued yet.
+        /// </summary>
+        /// <param name="timeStamp">The timestamp that is being checked.</param>
+        /// <returns>The status for if we want to queue the passed timestamp.</returns>
+        public static bool TimeStampReadyForQueue(float timeStamp) => 
+            timeStamp > time && timeStamp - (travelTime * 1.1) < time;
+
+        /// <summary>
+        /// Will update the difficulty through a newly passed difficulty.
+        /// </summary>
+        /// <param name="newDifficulty">The new difficulty.</param>
+        public static void SelectDifficulty(Difficulty newDifficulty) => difficulty = newDifficulty;
```

Comment:
What is the reason this is in a method? I think this method is unnecessary because all the fields in this script are `public` and accessible for other scripts.